nbs/data_augmentation_sequential.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.    img = cv2.imread("panda.jpg", cv2.IMREAD_COLOR)
</pre><p>can we start using/promoting kornia.io to load images ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kornia/tutorials/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I tried to modify as little as possible in these tutorials -- but because of the renaming it treated it as a completely new file when modified -- I had planned to update this later </p>